src/libstd/os/linux/raw.rs

```diff
@@ -47,7 +47,7 @@ mod arch {
     #[stable(feature = "raw_ext", since = "1.1.0")] pub type time_t = i64;
 
     #[repr(C)]
-    #[derive(Clone)]
+    #[derive(Clone, Debug)]
```

Comment:
These types are long since deprecated and otherwise this `Debug` is sort of just a portability hazard (got to remember to change *everything* in this folder). Perhaps these could be omitted?